composer.json

```
@@ -24,16 +22,10 @@
 		"source": "https://github.com/infinum/eightshift-boilerplate"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^v1.0.0",
-		"infinum/eightshift-coding-standards": "2.0.0-beta",
-		"php-stubs/wordpress-stubs": "^6.3",
-		"szepeviktor/phpstan-wordpress": "^1.3",
 		"wp-cli/wp-cli": "^2.9"
 	},
 	"require": {
-		"php": "^7.4 || >=8.0",
-		"infinum/eightshift-libs": "^7.0.0",
-		"typisttech/imposter-plugin": "^0.6.2"
+		"php": "^7.4 || >=8.0"
```

```suggestion
		"php": ">=8.0"
```
As this is a very major new version, I'd say we're very fine to ditch 7.4.
I'd even consider making 8.2 the min version.

---

☝️

---

done